docs/fedora-releases-guide.md

```diff
@@ -178,6 +178,13 @@ the changelog entry will use the GitHub/GitLab release description field.
 E.g. use `-` instead of `*` for lists to not create multiple changelog entries.)
 There is also [`sync_changelog`](/docs/configuration/#sync_changelog) configuration option to enable syncing 
 the whole changelog.
+You can also utilize a [custom `changelog-entry` action](/docs/configuration/actions#syncing-the-release).
+
+        actions:
+          changelog-entry:
+            - bash -c "echo '- New release' ${PACKIT_PROJECT_VERSION}"
```

Comment:
```suggestion
            - bash -c 'echo "- New release ${PACKIT_PROJECT_VERSION}"'
```

I would say this is easier to comprehend?

---

unless you start shoving in the variables; don't ask please…

Reply:
:+1: I'll update the other example I've copied this from as well..;)

Reply:
Fixed.